.github/workflows/ci.yml

```diff
@@ -6,34 +6,31 @@ on:
     pull_request:
 
 jobs:
-    build:
+    test:
         name: Build
         runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                node:
+                    # - 10
+                    # - 12
+                    # - 14
+                    - 15
```

Comment:
I see that at the moment other Node versions are commented, I'm just opening the discussion in advance: do we need to test with Node 12 and 14? Or can we target the lowest/highest (10/15) and be confident that it should work with the rest?

Reply:
All tests run in parallel, there is at the moment no additional cost brought by actually testing all node major between 10 and 15. Therefor I do not see any reason not to request it as part of the testing matrix :)